data/net/minecraft/world/level/levelgen/synth/ImprovedNoise.mapping

```diff
@@ -1,10 +1,47 @@
 CLASS net/minecraft/world/level/levelgen/synth/ImprovedNoise
+	COMMENT Generates a single octave of Perlin noise.
+	FIELD p [B
+		COMMENT A permutation array used in noise calculation.
+		COMMENT This is populated with the values [0, 256) and shuffled per instance of {@code ImprovedNoise}.
+		COMMENT
+		COMMENT @see #p(int)
```

Comment:
```suggestion
		COMMENT @see net.minecraft.world.level.levelgen.synth.ImprovedNoise#p(int)
```

Reply:
[See linked](https://github.com/ParchmentMC/Parchment/pull/39#discussion_r663562284). In short, this link will work without the FQN so it is unnecessary.

Reply:
As you mention in that link I thought that automatic cross-version migration was part of the reason for using FQN.